### ISSUE TITLE
Add SDRPP

### ISSUE THREAD

**Issue by fdlind** [ryanvolz/radioconda#79]
The SDRPP interface is pretty nice overall. Would make a nice addition. 

https://github.com/AlexandreRouma/SDRPlusPlus

**Comment by ryanvolz** [comment]
Yeah, I've already had my eyes on it but having it as an issue will be a helpful reminder. I think I've gotten to the point where conda-forge has most if not all of the dependencies now, so it's just a matter of packaging.

**Comment by dl5raz** [comment]
I'm waiting for this solution too!!
